src/components/Dashboard/FilingHistoryList/FutureEffective.vue

```diff
@@ -17,7 +17,7 @@
 
     <p>Registries contact information:</p>
 
-    <contact-info class="pt-3" />
+    <ContactInfo class="mt-4" />
```

Comment:
I made this consistent in all detail components. I tested a few of these and they look good.